src/ovirtapi/index.js

```diff
@@ -163,7 +164,7 @@ const OvirtApi = {
     return httpGet({ url })
   },
 
-  addNewVm ({ vm, transformInput = false }: VmType): Promise<Object> {
+  addNewVm ({ vm, transformInput = false }: { vm: VmType | Object, transformInput?: boolean }): Promise<ApiVmType> {
```

Comment:
Why do you have here ` | Object`. `vm` parameter must contain information, no? In any case I'd like to avoid `Object`, because this is not very good practice.

Reply:
I'll take another look at the `VmType`s on the API functions

Reply:
`vm` can take the type `Object` when `transform: false` and the direct contents of `vm` is sent to the API.